.github/workflows/build.yaml

```diff
@@ -81,6 +56,14 @@ jobs:
 
   clang_tidy:
     runs-on: ubuntu-latest
+    # Set job outputs to values from changed_files step for other jobs
+    outputs:
+      run_test_suite: ${{ steps.changed_files.outputs.test_suite }}
+      create_installer: ${{ steps.changed_files.outputs.installer }}
+      inst_windows_msvc: ${{ steps.changed_files.outputs.inst_windows_msvc }}
+      inst_windows: ${{ steps.changed_files.outputs.inst_windows }}
+      inst_macos: ${{ steps.changed_files.outputs.inst_macos }}
+      appimage: ${{ steps.changed_files.outputs.appimage }}
```

Comment:
Yes, it needs job output because a job can not access the steps of another job.
(see https://docs.github.com/en/actions/using-jobs/defining-outputs-for-jobs)

Reply:
Separation can be done in this way.
My previous strategy was to not add an additional job to not make full runs longer. But I just found out the action "dorny/paths-filter" can run without checkout (which takes about 15 s), so the extra time is only starting and stopping one job plus triggering the dependent ones, which should be very short.